### PR TITLE
Adds `zirco` command

### PR DESCRIPTION
**Pull request by ghost**
Hard coded the Zirco Dice service address and uses pawer withdraw service.
- Validates OpenField data i.e, 'odd' or 'even'
- Max bet amount limited to 100 as per ZircoDice spec.

Usage : 
`Pawer zirco <amount> <odd/even>`

### CODE REVIEW

File: cogs/bismuth.py [hunk 1]
```diff
@@ -348,6 +348,61 @@ def __init__(self, bot):
...
+                    await self.bot.add_reaction(ctx.message, '👍')  # Thumb up
+                    await self.bot.say("Your bet has been placed. Txid is {}".format(txid))
+                else:
+                    await self.bot.add_reaction(ctx.message, '👎')  # Thumb down
+                    await self.bot.say("Can't place your bet. Error {}".format(send['error']))
+                return
+            # Depending on channel, say or send PM
+            em = discord.Embed(description=DISCLAIMER, colour=discord.Colour.red())
+            em.set_author(name="You have to create your address first:")
+            await self.bot.say(embed=em)
```
Comment (EggPool) [inline cogs/bismuth.py]: I think this is not needed there

File: cogs/bismuth.py [hunk 2]
```diff
@@ -348,6 +348,61 @@ def __init__(self, bot):
...
+                return
+
+            if user_info and user_info['address']:
+                # User exists and validated the terms, has an address
+                # Make sure balance is enough
+                balance = float(user.balance())
+                msg = "{} withdraw {}, balance is {} ".format(ctx.message.author.display_name, amount, balance)
+                fees = BismuthUtil.fee_for_tx(bet)
+                print(msg)
+                if balance < amount + 0.01:
```
Comment (EggPool) [inline cogs/bismuth.py]: Fees are in fact 0.01003 for "odd" and 0.01004 for "even".
Use 0.01004 then.

File: cogs/bismuth.py [hunk 3]
```diff
@@ -348,6 +348,61 @@ def __init__(self, bot):
...
+                print("Bet amount too high")
+                await self.bot.add_reaction(ctx.message, '😟')
+                await self.bot.say("You are betting too high. Recommended amount is less than 100`")
+                return
+
+            if user_info and user_info['address']:
+                # User exists and validated the terms, has an address
+                # Make sure balance is enough
+                balance = float(user.balance())
+                msg = "{} withdraw {}, balance is {} ".format(ctx.message.author.display_name, amount, balance)
```
Comment (EggPool) [inline cogs/bismuth.py]: zirco, not withdraw

**Review by EggPool — COMMENTED** [review]
A few comments to address before merging.

**Comment by EggPool** [comment]
Awesome, merging!

**Comment by hclivess** [comment]
Amazing, thank you

**Pull request merged**